classes/Equipment.py

```diff
@@ -0,0 +1,22 @@
+from exceptions.ClassInitializationError import *
+
+
+class Equipment:
+    __parts = []
+    __name = ''
+
+    def __init__(self, name):
+        if isinstance(name, str):
+            self.__name = name
+        else:
+            raise ClassInitializationError('Name must be string')
+
+    def get_parts_info(self):
+        parts = self.__parts
+        return parts
+
+    def add_part(self, parts):
+        if isinstance(parts, list):
+            self.__parts += parts
```

Comment:
We can't check what contains a list accepted in this method. The main idea is checking if the provided list contains only objects implemented as SparePart.
Unfortinataly we have the same problem in the SparePart class. Please fix it too

Reply:
Fix